src/Script/Composer/Composer.php

```diff
@@ -42,7 +39,8 @@ public static function removeUnusedServices(
     public static function buildServiceMapping()
     {
         $serviceMapping = [];
-        $source = Aws\manifest();
+        $manifest = __DIR__ . '/../../data/manifest.json.php';
```

Comment:
Do we need the definition of the variable `$mainfest`?

Reply:
I threw this PR up pretty quickly and (without checking) thought the combination of the two lines might throw us over the PSR-recommended line length.  Looks like it's well below the max recommended line length (63 char).  Nice catch